Source/gs/GSH_OpenGL/GSH_OpenGL.cpp

```diff
@@ -1833,6 +1890,27 @@ void CGSH_OpenGL::CopyToFb(
 
 void CGSH_OpenGL::WriteRegisterImpl(uint8 nRegister, uint64 nData)
 {
+	switch(nRegister)
+	{
+	case GS_REG_TEX0_1:
+	case GS_REG_TEX0_2:
+	{
+		// If we use a CLUT based texture, its possible that the memory
```

Comment:
Would it work to put that logic inside a new SyncCLUT function inside the `CGSH_OpenGL` (that overrides the base function from `CGSHandler`)?

Reply:
Done